### PR TITLE
Fix setYPosition to avoid useless state change

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

**Comment by sroy3** [comment]
You might want to add a UT for that.

File: src/components/actions/ActionBarReducers.ts [hunk 1]
```diff
@@ -32,10 +32,12 @@ export const actionBarReducer = (state: IActionBarState = actionBarInitialState,
...
-            return {
-                ...state,
-                tableYPosition: action.payload.yPosition,
-            };
+            if (state.tableYPosition !== action.payload.yPosition) {
+                return {
+                    ...state,
+                    tableYPosition: action.payload.yPosition,
+                };
+            }
```
[the remark this answers is not in the document]
Comment (pochretien) [reply src/components/actions/ActionBarReducers.ts]: nope, I will fallback in the default case :D

[the remark this answers is not in the document]
Comment (MichelML) [reply src/components/actions/ActionBarReducers.ts]: not that I care that much, and PO is right, but does a switch case will perform a checkup for all the cases left before falling back on default ?  Secondly, i wonder if it's not a bit more concise the other way we do it:  
  
```  
return state.tableYPosition !== action.payload.yPosition  
    ? {...state, tableYPosition: action.payload.tableYPosition}
    : state;  
```

[the remark this answers is not in the document]
Comment (MichelML) [reply src/components/actions/ActionBarReducers.ts]: this was there before you, but shouldn't you make sure the payload.id is equal to the id in the state also ?

**Pull request merged**